packages/plugin-core/src/constants.ts

```diff
@@ -982,6 +982,13 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     desc: "Dump internal state of Dendron inside logs",
     docs: "This is useful when diagnosing issues in Dendron",
   },
+  DEV_TRIGGER: {
+    key: "dendron.dev.devTrigger",
+    title: `${CMD_PREFIX}Dev: Dev Trigger`,
+    desc: "Command to use for development to trigger some arbitrary piece of code.",
+    group: "dev",
+    skipDocs: true,
```

Comment:
can you add `when: DendronContext.DEV_MODE`?

Reply:
also added docs in https://github.com/dendronhq/dendron-docs/blob/main/vault/pkg.plugin-core.dev.cook.md#L22:L22

Reply:
Added the command to only be visible in command palette when `when: DendronContext.DEV_MODE`